### PR TITLE
new style for clay-starter sample article

### PR DESCRIPTION
**Pull request by jeanrodriguez**
- added style, css to clay starter sample-article.html 

- fallow https://app.zeplin.io/project/5c6f153327f5b4405c6ce26c/screen/5c6f155c6d6dc440975b3bda 
for design.

screen shot 2019-02-22 at 4 44 57 pm

### CODE REVIEW

File: app/styleguides/_default/common/_vars.css [hunk 1]
```diff
@@ -7,10 +7,14 @@ $sans-serif-narrow-stack: GroteskNarrow, sans-serif;
 
 /* Colors */
 
-$background-primary-color: #ebebeb;
+$background-primary-color: #ffffff;
```
Comment (Shilario) [inline app/styleguides/_default/common/_vars.css]:
```suggestion
$background-primary-color: #fff;
```

File: app/styleguides/_default/components/article.css [hunk 2]
```diff
@@ -26,18 +27,18 @@ $body-stack: Georgia, serif;
...
     }
 
     .headline-primary:last-child {
       margin: 0;
     }
 
     .bylines {
-      color: #808080;
-      letter-spacing: 1px;
+        font-size: 14px;
```
Comment (Shilario) [inline app/styleguides/_default/components/article.css]: fix indentation please

File: app/styleguides/_default/components/article.css [hunk 3]
```diff
@@ -46,18 +47,18 @@ $body-stack: Georgia, serif;
...
-    background: #EBEBEB;
+  .lede-image-wrapper {
+    margin-bottom: 20px
   }
 
   .lede-image-wrapper {
     img {
       display: block;
       width: 100%;
+      margin-bottom: 7px;
```
Comment (Shilario) [inline app/styleguides/_default/components/article.css]: Please Alphabetize the properties

File: app/styleguides/_default/components/article.css [hunk 4]
```diff
@@ -191,9 +183,11 @@ $body-stack: Georgia, serif;
...
     }
-
-    .body {
-      margin: 40px 0 0 100px;
-    }
   }
 }
+
+.article .attribution {
+  font-size: 12px;
```
Comment (Shilario) [inline app/styleguides/_default/components/article.css]: Please Alphabetize the properties

File: app/styleguides/_default/layouts/layout.css [hunk 5]
```diff
@@ -56,7 +56,8 @@ html {
 
   & > .page-header,
   & > .top {
-    width: 100%;
+    width: $wrapperMaxWidth;
```
Comment (Shilario) [inline app/styleguides/_default/layouts/layout.css]: same here

File: app/styleguides/_default/layouts/layout.css [hunk 6]
```diff
@@ -56,7 +56,8 @@ html {
...
+    margin:0 auto;
```
Comment (Shilario) [inline app/styleguides/_default/layouts/layout.css]:
```suggestion
    margin: 0 auto;
```

File: app/styleguides/_default/layouts/layout.css [hunk 7]
```diff
@@ -111,24 +112,24 @@ html {
...
       display: flex;
       justify-content: flex-end;
-      width: 1100px;
-    }
-
-    & > .wrapper > .main {
-      margin: 0 100px 0 0;
-      width: 600px;
-      z-index: 1;
+      max-width: $wrapperMaxWidth;
```
Comment (Shilario) [inline app/styleguides/_default/layouts/layout.css]: same here

File: app/styleguides/_default/layouts/layout.css [hunk 8]
```diff
@@ -111,24 +112,24 @@ html {
...
 
     &.kiln-edit-mode .wrapper > .main {
-      width: 700px;
+      width: 100%;
     }
   }
 }
+
+.wrapper {
+    width: 100%;
```
Comment (Shilario) [inline app/styleguides/_default/layouts/layout.css]: same here

**Comment by Shilario** [comment]
Also needs a rebase

File: app/styleguides/_default/layouts/layout.css [hunk 9]
```diff
@@ -48,29 +48,30 @@ html {
...
-    width: 100%;
+    margin: 0 auto;
+    width: $wrapperMaxWidth;
   }
 
-  & > .primary,
-  & > .secondary,
-  & > .bottom,
-  & > .wrapper {
+  & > .bottom{
```
Comment (Shilario) [inline app/styleguides/_default/layouts/layout.css]:
```suggestion
  & > .bottom {
```

File: app/styleguides/_default/layouts/layout.css [hunk 10]
```diff
@@ -48,29 +48,30 @@ html {
...
     width: 100vw;
   }
+  & > .wrapper{
```
Comment (Shilario) [inline app/styleguides/_default/layouts/layout.css]:
```suggestion
  & > .wrapper {
```

File: app/styleguides/_default/components/article.css [hunk 11]
```diff
@@ -46,24 +47,25 @@ $body-stack: Georgia, serif;
...
 
     .lede-image-data {
-      background: #EBEBEB;
       position: relative;
     }
   }
 
   .attribution {
-    font: bold 13px/16px $body-stack;
+    color: #767676;
```
Comment (manuelurenah) [inline app/styleguides/_default/components/article.css]: This can be set as a variable variable at the top of the fiel

**Pull request merged**